code/modules/vehicles/cars/car.dm

```diff
@@ -1,18 +1,34 @@
 /obj/vehicle/sealed/car
 	layer = ABOVE_MOB_LAYER
 	anchored = TRUE
+	default_driver_move = FALSE
 	var/car_traits = NONE //Bitflag for special behavior such as kidnapping
 	var/engine_sound = 'sound/vehicles/carrev.ogg'
 	var/last_enginesound_time
 	var/engine_sound_length = 20 //Set this to the length of the engine sound
 	var/escape_time = 200 //Time it takes to break out of the car
 
+/obj/vehicle/sealed/car/Initialize()
+	. = ..()
+	var/datum/component/riding/D = LoadComponent(/datum/component/riding)
+	D.vehicle_move_delay = 2
```

Comment:
It's the tick delay between movements. Don't see what you mean by magic numbers. I actually don't even need this here at all though because for some reason I thought the default in the component was 1. I'll just get rid of it :^)

Reply:
magic numbers as in the 2 is a random number thats not being pulled from anywhere, for example I imagine we want the vehicle_move_delay to be our vehicles move delay, instead of 2.

Reply:
Oh yeah, I understand what you mean. I'll change it.

Reply:
you clearly didnt